docs/csharp/language-reference/keywords/value-types.md

```diff
@@ -37,16 +27,30 @@ The value types consist of two main categories:
   
  Unlike reference types, a value type cannot contain the `null` value. However, the [nullable types](../../../csharp/programming-guide/nullable-types/index.md) feature does allow for value types to be assigned to `null`.  
   
- Each value type has an implicit default constructor that initializes the default value of that type. For information about default values of value types, see [Default Values Table](../../../csharp/language-reference/keywords/default-values-table.md).  
-  
-## Main Features of Simple Types  
- All of the simple types -- those integral to the C# language -- are aliases of the .NET Framework System types. For example, [int](../../../csharp/language-reference/keywords/int.md) is an alias of <xref:System.Int32?displayProperty=nameWithType>. For a complete list of aliases, see [Built-In Types Table](../../../csharp/language-reference/keywords/built-in-types-table.md).  
-  
- Constant expressions, whose operands are all simple type constants, are evaluated at compilation time.  
+ Each value type has an implicit default constructor that initializes the default value of that type. For information about default values of value types, see [Default values table](default-values-table.md).  
   
- Simple types can be initialized by using literals. For example, 'A' is a literal of the type `char` and 2001 is a literal of the type `int`.  
+## Simple types
```

Comment:
- It seems like all of the following text should go into the structs topic not the more general value types topic. All of this is specific to structs, right?
- The "simple types" section seems to beg for a matching user-defined types section.

Reply:
> The "simple types" section seems to beg for a matching user-defined types section

What would such a section contain? Is there any property of user-defined structs that is not valid for a simple type?

> It seems like all of the following text should go into the structs topic not the more general value types topic. All of this is specific to structs, right?

Though I agree with this, I don't think this PR should handle it. Simply because what is the structs topic? There are several candidates:
- https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/struct
- https://docs.microsoft.com/en-us/dotnet/csharp/structs
- https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/structs

#966 is tracking the duplication issue.

Reply:
> What would such a section contain? Is there any property of user-defined structs that is not valid for a simple type?

There may not be differences. It's more that there is a section dedicated to one form of structs and not to the other.

> Though I agree with this, I don't think this PR should handle it.

Sounds good.

Reply:
> It's more that there is a section dedicated to one form of structs and not to the other.

I see the whole picture as one set of structs (simple types) is a subset of the set of all struct types. So, what should be described is (1) structs in general; (2) simple types. Though, it's good idea to say that a user-defined struct is a struct, which is not a simple type. I don't know where to put it yet in docs. So, come back to it later (together with another review of this topic, which can still be improved).